src/views/templates/list/hooks/useWorkloadProfile.ts

```diff
@@ -6,18 +6,19 @@ import { getTemplateWorkload } from '../../utils/selectors';
 // t('plugin__kubevirt-plugin~Desktop')
 // t('plugin__kubevirt-plugin~Server')
 // t('plugin__kubevirt-plugin~High performance')
-// t('plugin__kubevirt-plugin~Other')
 
-const workloadLabels = {
+export const workloadLabels = {
   desktop: 'Desktop',
   server: 'Server',
   highperformance: 'High performance',
 };
 
-const useWorkloadProfile = (template: V1Template): string => {
+export type WorkloadProfileNames = 'Desktop' | 'Server' | 'High performance';
+
+const useWorkloadProfile = (template: V1Template): WorkloadProfileNames => {
```

Comment:
can we do the translation in the object that use this method ?

e.g. 
the consumer run `t`
```tsx
const useWorkloadProfile = (template) => getVmObj(template).spec.template.metadata.annotations['vm.kubevirt.io/workload']
<some component label={t(useWorkloadProfile(template))}>
```
vs this method run `t`
```tsx
const useWorkloadProfile = (template) => t(getVmObj(template).spec.template.metadata.annotations['vm.kubevirt.io/workload'])
<some component label={useWorkloadProfile(template)}>
```